src/pages/components/tooltip/usage.mdx

```diff
@@ -71,27 +79,43 @@ in tooltips; use the toggletip component instead.
 </Column>
 </Row>
 
-<Caption>Image of a tooltip</Caption>
-
 ### When to use
 
-- To expose names of controls, like icon buttons, that lack visual labels
-- When an element can take focus and supplying additional information is useful
-  in helping a user make decisions
-- When an element needs more context or explanation
-- Use when defining a term or inline item
+By default, most components such as icon buttons, icon tabs, sliders, and copy
+buttons already have built-in tooltips, so there’s no need to add one
+separately. Customization is allowed but not recommended.
+
+Tooltips can be used to:
```

Comment:
Should this be an H4? Could go either way.

Reply:
Yeah, the intention is that it's still connected to the first paragraph so I'd leave it as the body text.